test/blade/BaseBridgeGateway.test.ts

```diff
@@ -0,0 +1,135 @@
+import { expect } from "chai";
+import * as hre from "hardhat";
+import { ethers } from "hardhat";
+import { BLS, BN256G2, BaseBridgeGateway } from "../../typechain-types";
+import * as mcl from "../../ts/mcl";
+import { bridge } from "../../typechain-types/contracts";
+
+const DOMAIN = ethers.utils.arrayify(ethers.utils.solidityKeccak256(["string"], ["DOMAIN_BRIDGE"]));
+
+describe("BridgeStorage", () => {
```

Comment:
BaseBridgeGateway instead of BridgeStorage?

Reply:
Is this comment addressed?